lib/cdo/shared_constants.rb

```diff
@@ -652,6 +652,33 @@ module SharedConstants
     }
   ).freeze
 
+  VOICES = {
```

Comment:
Resurrecting my old comment from the pre-LFS PR: Why are we putting these into `shared_constants.rb` if none of these constants are referenced in Rails?

Reply:
This was because we had duplicated the `VOICES` in both JavaScript and Ruby (in `dashboard/app/models/concerns/text_to_speech.rb`) and that's what `SharedConstants` are for.

What is missing is using the `SharedConstants` version of `VOICES` in the `TextToSpeech` concern and removing it from there.

Reply:
Oh ok, makes total sense. We should probably open a ticket to do that, otherwise we haven't really improved the situation by moving them into shared_constants.